src/Documentation/Markdown/Markdown.js

```diff
@@ -104,6 +104,27 @@ CodeBlock.propTypes = {
   value: PropTypes.node.isRequired
 }
 
+const Link = ({ children, ...props }) => {
+  const externalLink = props.href.match(/^(\/\/|http(s)?:\/\/)/)
+  const showIcon =
+    externalLink && children && typeof children[0].props.children === 'string'
+
+  const modifiedProps = externalLink
+    ? { ...props, target: '_blank', rel: 'noopener nofollow' }
```

Comment:
What if props already has a `target`?

Reply:
Right now we will rewrite it with `_blank` value and it's probably ok.

Other possible values are: `_self`, `_parent`,  `_top` and `framename`. `_parent` and `_top` will not work anyway. `framename` can work in some situations, but we don't have any frames right now (and it didn't sound secure to me to open external links in the iframes on our site). `_self` is somehow useful if we want to allow to manually prevent some external links from opening in the new window, but I'd want for real usecase examples before enabling it.

Reply:
So if I write <a href="..." target="_self">link</a> it will overwrite with `_target`? I think it would be god to at least note this in the README to prevent someone in the future spending hours trying to figure out this one line of code :P

Reply:
> So if I write <a href="..." target="_self">link</a> in Markdown

you probably should not be doing this though in Markdown (does it work with the Remark at all?). But I agree, may be we should check and do not overwrite it if it's set explicitly.

Reply:
Markdown supports all HTML AFAIK. It's necessary for specific tag params like `<a target`

Reply:
> does it work with the Remark at all?

Hm.. I tried and actually `<a>` tags don't get rendered as links for some reason. We use `<abbr>` and `<details>` a lot so I don't understand this behavior.

Reply:
there are custom handlers for <abbr> and <details>. In general it's better to avoid using any HTML in Markdown

Reply:
So if our Markdown doesn't even support the `<a>` tag, probably this is OK and there's no need to check for `target` values before overwriting? If you agree please just resolve this.

Reply:
resolved! if @iAdramelk decide that we need to check it we'll add it later. I think we are good for now.